io_scene_godot/converters/simple_nodes.py

```diff
@@ -145,8 +172,13 @@ def export_lamp_node(escn_file, export_settings, node, parent_gd_node):
 
         # Properties common to all lights
         light_node['transform'] = fix_directional_transform(node.matrix_local)
```

Comment:
could you do me a favor adding a comment here just mention these two attributes are specially handled, because they are not the same format in node attribute and animation value, really appreciate it!